src/styles/themes/default.js

```diff
@@ -60,6 +60,9 @@ const darkTheme = {
     checkBox: colors.green,
     pickerOptionsTextColor: colors.white,
     imageCropBackgroundColor: colors.greenIcons,
+
+    // Brand color keys
+    green700: colors.green700,
```

Comment:
NAB: I wonder if the key `green700` should be named after what is being used for? like... `fallbackIconColor` or something